meson.build

```diff
@@ -0,0 +1,55 @@
+project('nnstreamer', 'c', 'cpp',
+    version: '0.0.3',
+    license: ['LGPL'],
+    meson_version: '>=0.29.0',
```

Comment:
As well all know, Meson runs based on the Ninja package. Note that we have to append "BuildRequires: ninja" additionally for the Meson build in ./packaging/ (for Tizen) and ./debian/ (for Ubuntu).